### PR TITLE
dts: fix warnings in nxp_rt11xx.dtsi

### LINKED ISSUE #76155: mimxrt11xx: Compiler warning after recent commit regarding 'enable mcux udc on RT1170-EVK'
With Zephyr 3.7.0-rc3, I now get this warning when compiling for board mimxrt1170_evk_cm7 (if it matters, I was compiling the openamp sample, but it also applies to basic/blinky):

> C:/iseg_nla/zephyrproject/zephyr/samples/subsys/ipc/openamp/build/openamp_remote/zephyr/zephyr.dts:3229.30-3233.5: Warning (simple_bus_reg): /soc/usbphy@0x40434000: simple-bus unit address format error, expected "40434000"
> C:/iseg_nla/zephyrproject/zephyr/samples/subsys/ipc/openamp/build/openamp_remote/zephyr/zephyr.dts:3234.30-3238.5: Warning (simple_bus_reg): /soc/usbphy@0x40438000: simple-bus unit address format error, expected "40438000"

The reason is due to a slight error in [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/9a45300ab50050380a98f7167d0f476efcc74245), I think. 

In nxp_rt11xx.dtsi, @MarkWangChinese added these lines:
```
		usbphy1: usbphy@0x40434000 {
			compatible = "nxp,usbphy";
			reg = <0x40434000 0x1000>;
			status = "disabled";
		};

		usbphy2: usbphy@0x40438000 {
			compatible = "nxp,usbphy";
			reg = <0x40438000 0x1000>;
			status = "disabled";
		};
```

But the correct syntax is usbphy@40434000 WITHOUT 0x.

### PR DESCRIPTION
**Pull request by Finomnis**
Building a project based on the rt11xx produces the following warning:

```
Warning (simple_bus_reg): /soc/usbphy@0x40434000: simple-bus unit address format error, expected "40434000"
Warning (simple_bus_reg): /soc/usbphy@0x40438000: simple-bus unit address format error, expected "40438000"
```

It is caused by a typo in the device tree, which this PR fixes.

### CODE REVIEW

**Review by henrikbrixandersen — APPROVED** [review]

**Review by decsny — APPROVED** [review]
I noticed this too but forgot to fix it when I saw it

**Comment by Finomnis** [comment]
> I noticed this too but forgot to fix it when I saw it

What did I do wrong? How do I make CI happy?

**Review by iuliana-prodan — APPROVED** [review]

**Comment by Finomnis** [comment]
@decsny What's missing? Do I need to do something?

**Comment by decsny** [comment]
> @decsny What's missing? Do I need to do something?

No, just waiting for the release team

**Review by dleach02 — APPROVED** [review]

**Comment by Finomnis** [comment]
@dleach02 Now that 3.7 is out, could this change be released with 3.7.1?

**Pull request merged**
